low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -119,16 +120,14 @@ def read_stored_cpu_info(self):
             if self.stored_cpu_info is not None and self._node_id in self.stored_cpu_info.keys():
                 self.prev_cpu_info = self.stored_cpu_info[self._node_id]['CPU_LIST']
         except Exception as e:
-            logger.error(f"Error while reading stored cpu info, shutting down CPUFaultSensor : {e}")
-            self.shutdown()
+            raise Exception(f"Error while reading stored cpu info, {e}")
```

Comment:
this won't help us to find where the actual exception happened. pls use
`raise Exception(f"Error while reading current cpu info") from e`

Reply:
Please refer thread_controller.py for capturing and logging full traceback of the error.